templates/course_form/course_form.html

```diff
@@ -11,53 +11,86 @@
 <body>
     {% include 'navigation_bar/navigation.html' %}
     <div class="container">
-        <h1>Course Form</h1>
-        <form method="post" action="">
-            {% csrf_token %}
-            <div class="form-group">
-                <label for="course_code">Course Code</label>
-                <input
-                    type="text"
-                    id="course_code"
-                    name="course_code"
-                    value="{{ data.code }}"
-                    {% if isEditing %}readonly{% endif %}
-                >
-            </div>
-            <div class="form-group">
-                <label for="semester">Semester</label>
-                <select name="semester">
-                    {% for semester in existSemester %}
-                        <option value={{ semester.semester_name }}>{{ semester.semester_name }}</option>
-                    {% endfor %}
-                </select>
-            </div>
-            <div class="form-group">
-                <label for="course_name">Course Name</label>
-                <input
-                    type="text"
-                    id="course_name"
-                    name="course_name"
-                    value="{{ data.name }}"
-                >
-            </div>
-
-            <!-- Form Actions -->
-            <div class="form-actions">
-                <button type="submit">Save</button>
-                <a href="{% url 'home' %}" class="cancel-link">Cancel</a>
-            </div>
-        </form>
-
-        <!-- Delete Section Form -->
-        {% if data.is_deletable %}
-        <form method="post" action="{% url 'section-form' code=course.course_code semester=semester section_number=data.section_number %}">
+    <h1>Course Form</h1>
```

Comment:
the indent level is still a little off. Look at the diff in git to see what I mean.